bambinomio/index.html

```diff
@@ -2,78 +2,106 @@
-  <link rel="stylesheet" href="./assets/css/bambino.color.themes.css">
-  <link rel="stylesheet" type="text/css" href="./assets/css/gallery.css" />
-  <link rel="stylesheet" type="text/css" href="./assets/css/page.css" />
-  <!-- javascript -->
-  <script src="https://www.w3schools.com/lib/w3.js"></script>
-  <script src="assets/js/includeHTML.js"></script>
-  <script src="assets/js/gallery.js"></script>
-  <script src="assets/js/basket.js"></script>
-  <script src="assets/js/navigation.js"></script>
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-
-  <script src="assets/js/data/simpleProducts.js"></script>
-  <script src="assets/js/data/swimnappies.js"></script>
-  <script src="assets/js/data/miosoft.js"></script>
-  <script src="assets/js/data/miosolo.js"></script>
+    <title>bambinomio.lv</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- css -->
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./assets/css/bambino.color.themes.css">
+    <link rel="stylesheet" type="text/css" href="./assets/css/gallery.css" />
+    <link rel="stylesheet" type="text/css" href="./assets/css/page.css" />
+    <!-- javascript -->
+    <script src="https://www.w3schools.com/lib/w3.js"></script>
+    <script src="assets/js/includeHTML.js"></script>
+    <script src="assets/js/gallery.js"></script>
+    <script src="assets/js/basket.js"></script>
+    <script src="assets/js/navigation.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+    <script src="assets/js/data/simpleProducts.js"></script>
+    <script src="assets/js/data/swimnappies.js"></script>
+    <script src="assets/js/data/miosoft.js"></script>
+    <script src="assets/js/data/miosolo.js"></script>
 </head>
 
 <body>
 
-  <!-- navigation -->
-  <div w3-include-html="assets/html/navigation.html"></div>
-
-  <!-- introduction -->
-  <div class="w3-text-theme w3-panel w3-content w3-padding-64" style="max-width:2000px">
-    <div class="w3-row">
-      <div class="w3-container w3-quarter ">
-        <img src="assets/img/logo.png" class="w3-round w3-margin-bottom" alt="logo" style="width:60%">
-      </div>
-      <div class="w3-container w3-threequarter ">
-        <ul class="w3-ul w3-center ">
-          <li>
-            <h2>atkārtoti lietojamie autiņi</h2>
-          </li>
-          <li>
-            <h2>draudzīgs bērnam</h2>
-          </li>
-          <li>
-            <h2>draudzīgs videi</h2>
-          </li>
-          <li>
-            <h2>draudzīgs makam</h2>
-          </li>
-        </ul>
-      </div>
+    <div class="w3-top">
+        <div class="w3-row w3-theme-d3">

```

Comment:
W3-row izraisīs "responsive design" uzvedību. Vai to gribam šeit? T.i. - vai gribam, ka atsevišķi bloki sāk sastāties viens zem otra, kad viegli samazinām ekrāna izmēru? Īpaši - zemāk izmantojam w3-bar. Atšķirībā no w3-row w3-bar nesāks "sabrukt daļās", kad ekrāna izmērs samazināsies. 

Es varu iedomāties, ka iekš w3-bar varētu likt w3-row, varbūt (neesmu drošs). 
Bet likt w3-bar iekš w3-row - tas man šķiet uzprasīties uz nepatikšanām. 

Ja esam droši, ka tām rindām nevajag sašķelties daļās (kā nekā šis ir variants tikai lielajam ekrānam) , varbūt visur labāk pieturēties pie w3-bar?